mcs/class/System.Drawing/System.Drawing.dll.sources

```diff
@@ -221,3 +221,12 @@ System.Drawing.Text/PrivateFontCollection.cs
 ../../../external/corefx/src/System.Drawing.Common/src/System/Drawing/Text/InstalledFontCollection.cs
 ../../../external/corefx/src/System.Drawing.Common/src/System/Drawing/Text/TextRenderingHint.cs
 
+../../../external/corefx/src/Common/src/System/Drawing/ColorTable.cs
+../../../external/corefx/src/Common/src/System/Drawing/ColorUtil.netcoreapp20.cs
+
+../../../external/corefx/src/Common/src/Interop/Unix/Interop.Libraries.cs
+../../../external/corefx/src/Common/src/Interop/Unix/libdl/Interop.dlopen.cs
+../../../external/corefx/src/Common/src/Interop/Unix/libdl/Interop.dlsym.cs
+
+../../../external/corefx/src/System.Drawing.Common/src/System/Drawing/Drawing2D/CustomLineCap.Unix.cs
+../../../external/corefx/src/System.Drawing.Common/src/System/Drawing/GdiplusNative.Unix.cs
```

Comment:
this will break on OSX and Windows since it tries to `dlopen("libgdiplus.so")`.

We actually want to continue to use our p/invoke dllmap to take effect for gdiplus, though. 

Do you know if there's a way to get a handle to the native library from a p/invoke function? This would allow us to add a normal p/invoke call to some gdiplus function so the runtime looks up dllmap etc instead of the explicit `dlopen()` and then just pass that handle to the subsequent `dlsym()` in the corefx code.

Reply:
What do we what dllmap for here?